docs/azure_arc_jumpstart/azure_arc_k8s/day2/calico/_index.md

```diff
@@ -0,0 +1,127 @@
+
+## Prerequisites
+
+* Clone the Azure Arc Jumpstart repository
+
+    ```shell
+    git clone https://github.com/microsoft/azure_arc.git
+    ```
+
+* [Install or update Azure CLI to version 2.25.0 and above](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest). Use the below command to check your current installed version.
+
+  ```shell
+  az --version
+  ```
+
+* As mentioned, this guide starts at the point where you already have a connected EKS/GKE cluster to Azure Arc.
+
+    ![Existing EKS Azure Arc connected cluster](./arcdemo-eks.png)
+
+    ![Existing GKE Azure Arc connected cluster](./arcdemo-gke.png)
+
+* Before installing the Azure Policy Add-on or enabling any of the service features, your subscription must enable the Microsoft.PolicyInsights resource provider and create a role assignment for the cluster service principal. To do that, open [Azure Cloud Shell](https://shell.azure.com/) and run either the Azure CLI or Azure PowerShell command.
+
+    ![Open Azure Cloud Shell](./03.png)
+
+    Azure CLI:
+
+    ```shell
+    az provider register --namespace 'Microsoft.PolicyInsights'
+    ```
+
+    Azure PowerShell:
+
+    ```powershell
+    Register-AzResourceProvider -ProviderNamespace 'Microsoft.PolicyInsights'
+    ```
+
+    To verify successful registration, run either the below Azure CLI or Azure PowerShell command.
+
+    Azure CLI:
+
+    ```shell
+    az provider show --namespace 'Microsoft.PolicyInsights'
+    ```
+
+    Azure PowerShell:
+
+    ```powershell
+    Get-AzResourceProvider -ProviderNamespace 'Microsoft.PolicyInsights'
+    ```
+
+    ![AzResourceProvider Bash](./04.png)
+
+
+* Azure Policy installed in Arc Connected Cluster
+
+ > by running the the ```kubectl get pods -n gatekeeper-system ``` command, you will notice all pods have been deployed.
+
+![Showing pods deployment](./05.png)
+
+
+## Deploy a customized Azure policy to verify if calicocloud/calico enterprise components enabled in your Azure Arc Kubernetes cluster.
+
+* In the Search bar, look for *Policy* and click on *Definitions* which will show you all of the available Azure policies.
+
+    ![Searching for Azure Policy definitions](./06.png)
+
+    ![Searching for Azure Policy definitions](./07.png)
+
+* Click on *+ Policy definition* to add a new policy. 
+
+    ![Create a Azure policy for Kubernetes cluster](./08.png)
+
+    ![Create a Azure policy for Kubernetes cluster](./09.png)
+
+* In the above example, the *Definition location* of the policy represent the Azure subscription. Also, make sure *Category* is set to *Kubernetes*. We will use `audit` as default effect for this policy. 
```

Comment:
```suggestion
* In the above example, the _Definition location_ of the policy represents the Azure subscription. Also, make sure _Category_ is set to _Kubernetes_. We will use `audit` as the default effect for this policy. 
```

Reply:
This sentence should come before 10.png IMO (if so, please update to "In the below example"

Reply:
move the sentence before 09.png